README_maintainers.md

```diff
@@ -642,209 +624,261 @@ These commands for `-c` mean:
 
     cdb -z <dump file> -y 'SRV*e:\symbol_cache*https://download.arangodb.com/symsrv_arangodbXX;SRV*e:\symbol_cache\ms*http://msdl.microsoft'
 
 and use the commands above to obtain stacktraces.
 
---------------------------------------------------------------------------------
+---
 
-Local cluster startup
-=====================
+Unittests
+---------
 
-There are two scripts `scripts/startLocalCluster` and
-`scripts/stopLocalCluster` which help you to quickly fire up a testing
-cluster on your local machine. `scripts/startLocalCluster` takes 0, 2 or
-three arguments. In the 0 argument version, it simply starts 2 DBservers
-and one coordinator in the background, running on ports 8629, 8630 and
-8530 respectively. The agency runs on port 4001. With 2 arguments the
-first is the number of DBServers and the second is the number of 
-coordinators.
+### Dependencies
 
-If there is a third argument and it is "C", then the first coordinator
-will be started with `--console`` in a separate window (using an
-`xterm`). 
+- *Ruby*, *rspec*, *httparty* to install the required dependencies run:
+  `cd tests/rb/HttpInterface; bundler`
+- *Google Test* (compile time, shipped in the 3rdParty directory)
 
-If there is a third argument and it is "D", then all servers are started
-up in the GNU debugger in separate windows (using `xterm`s). In that
-case one has to hit ENTER in the original terminal where the script runs
-to continue, once all processes have been start up in the debugger.
+### Folder Locations
 
-ArangoDB on Mesos
-=================
+There are several major places where unittests live:
 
-This will spawn a **temporary** local mesos cluster.
+| Path                         | Description
+|:-----------------------------|:-----------------------------
+| `tests/js/server/`           | JavaScript tests, runnable on the server
+| `tests/js/common/`           | JavaScript tests, runnable on the server & via arangosh
+| `tests/js/common/test-data/` | Mock data used for the JavaScript tests
+| `tests/js/client/`           | JavaScript tests, runnable via arangosh
+| `tests/rb/`                  | rspec tests (Ruby)
+| `tests/rb/HttpInterface/`    | rspec tests using the plain RESTful interface of ArangoDB. Include invalid HTTP requests and error handling checks for the server.
+| `tests/` (remaining)         | Google Test unittests
 
-Requirements:
+### Filename conventions
 
-- Somewhat recent linux
-- docker 1.10+
-- curl
-- sed (for file editing)
-- jq (for json parsing)
-- git
-- at least 8GB RAM
-- fully open firewall inside the docker network
+Special patterns in the test filenames are used to select tests to be executed
+or skipped depending on parameters:
 
-To startup a local mesos cluster:
+| Substring       | Description
+|:----------------|:----------------
+| `-server`       | Make use of existing external server. Example: `scripts/unittest http_server --server tcp://127.0.0.1:8529/`
```

Comment:
this is not a filename extension, but its rather an option to testing.js

Reply:
How does it know what kind of test to run? Is it a file name suffix as well as an option?

Reply:
it will work as before, without launching its own servers. Testsuites that fiddle with launching/stopping instances won't work with this.

Reply:
see https://github.com/arangodb/arangodb/pull/9507/files#diff-74605e757877535d6294a5e66a12da9dL681